app/experimenter/static/css/experimenter.css

```diff
@@ -85,10 +85,10 @@ select {
 }
 
 .heavy-line {
-   border: 0; 
-  height: 1px; 
-  background: #333; 
-  background-image: linear-gradient(to right, #ccc, #333, #ccc); 
+   border: 0;
```

Comment:
Somewhat unrelated, but the indenting here looks off by 1.

Reply:
Yeah, this change sucks. 

I only meant to add the lines 171-173 but my editor is configured to run `prettier` on every save. So it accidentally corrected this file. 

By the way, https://github.com/mozilla/experimenter/pull/957 takes care of this problem altogether.

Reply:
Please revert these style changes. Like I mentioned in #957, I don't think one-off style fixes like this are a good idea.